easy_thumbnails/management/commands/thumbnail_cleanup.py

```diff
@@ -53,7 +58,7 @@ def clean_up(self, dry_run=False, verbosity=1, last_n_days=0,
 
         query = Source.objects.all()
         if last_n_days > 0:
-            today = date.today()
+            today = datetime.today()
```

Comment:
There's a difference between `date.today()` and `datetime.today()`, the latter includes the current time as well (https://docs.python.org/3/library/datetime.html#datetime.datetime.today).

Not sure what the intent here is, but this change might have unintended results.

Reply:
Thanks for reviewing. This forced me to have a second look, I believe however, that my fixes do not add any regression.

Reply:
OK. Returned to previous logic.